dip-0028.md

```diff
@@ -229,37 +229,48 @@ type 1 requires a 4000 DASH collateral.
 
 #### Changes in ProRegTx
 
+> [!WARNING]
+> The field `platformInfo` is deprecated with the implementation of extended addresses as defined
+> in [Appendix C](dip-0003/network-info.md#extended-format) of DIP 0003. The information given below
+> is provided to document historical behavior.
+
 A ProRegTx with masternode type 1 indicates registration of a 4000 DASH collateral evonode.
 
-If the `version` is 2 and the `type` is 1, then the `platformNodeID`, `platformP2PPort`, and
-`platformHTTPPort` fields are serialized in that order between the `inputsHash` and `payloadSigSize`
-fields.
+If the `version` is 2 and the `type` is 1, then the `platformNodeID` and `platformInfo`
+fields are serialized in that order between the `inputsHash` and `payloadSigSize` fields.
 
 | Field | Type | Size | Description | Serialized |
 | --- | --- | --- | --- | --- |
 | platformNodeID | byte[] | 0 or 20 bytes | Dash Platform node ID, derived from P2P public key. Only present for masternode type 1. | Only when `version` is 2 and `type` is 1. |
-| platformP2PPort | uint_16 | 0 or 2 bytes | TCP port of Dash Platform peer-to-peer communication between nodes (network byte order). Only present for masternode type 1. | Only when `version` is 2 and `type` is 1. |
-| platformHTTPPort | uint_16 | 0 or 2 bytes | TCP port of Platform HTTP/API interface (network byte order). Only present for masternode type 1. | Only when `version` is 2 and `type` is 1. |
+| platformInfo | byte[] | variable | Platform network information (see [appendix](dip-0003.md#appendix-c-network-information)). Only present for masternode type 1. | Only when `version` is 2 and `type` is 1. |
```

Comment:
nit: `s/platformInfo/PlatformNetwork/` or something similar.
Because `platformNodeID` is also some kind of `info`, but here meant only network information.

Reply:
Wouldn't be opposed to renaming it to `platformNetInfo`, thoughts?